### PR TITLE
Script to update manifest.yaml and Changelog.md

### PR DESCRIPTION
**Pull request by ngama75**
- the initial release must be pushed manually to the main branch
- From now on, please use prepare-release to create a new release.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] - 2023-07-18
+
+- Initial release of the double precision fft on the reim and cplx backends
+- Coeffs-space conversions cplx <-> znx32 and tnx32
+- FFT-space conversions cplx <-> the reim4 layouts
```
Comment (sguaschsbt) [inline Changelog.md]: 'the'?

File: scripts/prepare-release [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+}
+print "Version in manifest file: $vmajor.$vminor.$vpatch\n";
+
+if (not -f $changefile) {
+	## create a changes file
+	open F,">$changefile";
+	print F "# Changefile for branch $currentbranch\n\n";
+	print F "## Type of release (major,minor,patch)?\n\n";
+	print F "releasetype: patch\n\n";
+	print F "## What has changed (please edit)?\n\n";
```
Comment (sguaschsbt) [inline scripts/prepare-release:72]: At some point we could get the PRs that were merged since the last release

File: scripts/prepare-release [hunk 3]
```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/perl
+##
+## This script will help update manifest.yaml and Changelog.md before a release
+## Any merge to master that changes release.yaml is considered as a new release
```
Comment (sguaschsbt) [inline scripts/prepare-release]: release.yaml or manifest.yaml?

File: scripts/prepare-release (hunk 2 above)
Comment (ngama75) [reply scripts/prepare-release:72]: On the other hand, we will try to release often, so hopefully, there won't be too much unreleased stuff to remember :)

**Review by sguaschsbt — APPROVED** [review]
looks good!

**Pull request merged**